### PR TITLE
Issue13

### PR DESCRIPTION
**Pull request by jeffwright13**
Support for gTTS "accents"

### CODE REVIEW

File: audio_program_generator/apg.py [hunk 1]
```diff
@@ -186,8 +202,11 @@ def main():
     sound_file = Path(args["<sound_file>"]) if args["<sound_file>"] else None
     slow = bool(args["--slow"])
     attenuation = int(args["--attenuation"]) if args["--attenuation"] else 0
+    tld = str(args["--tld"]) if args["--tld"] else "com"
+    tld = tld if tld in TLDs else "com"
+    print("tld:", tld)
```
Comment (bbelderbos) [inline audio_program_generator/apg.py:207]: f-string

File: audio_program_generator/apg.py [hunk 2]
```diff
@@ -186,8 +202,11 @@ def main():
     sound_file = Path(args["<sound_file>"]) if args["<sound_file>"] else None
     slow = bool(args["--slow"])
     attenuation = int(args["--attenuation"]) if args["--attenuation"] else 0
+    tld = str(args["--tld"]) if args["--tld"] else "com"
+    tld = tld if tld in TLDs else "com"
```
Comment (bbelderbos) [inline audio_program_generator/apg.py:206]: I usually default to a `set` for membership checking ( and if order does not matter)

**Review by bbelderbos — COMMENTED** [review]
Looks good

File: audio_program_generator/apg.py (hunk 1 above)
Comment (jeffwright13) [reply audio_program_generator/apg.py:207]: Yessir

**Pull request merged**